clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -569,6 +569,36 @@ def YieldOp : CIR_Op<"yield", [ReturnLike, Terminator,
   ];
 }
 
+//===----------------------------------------------------------------------===//
+// BreakOp
+//===----------------------------------------------------------------------===//
+
+def BreakOp : CIR_Op<"break", [Terminator]> {
+  let summary = "C/C++ `break` statement equivalent";
+  let description = [{
+    The `cir.break` operation is used to cease the execution of the current
+    loop or switch and transfer control to the parent operation. It is only
```

Comment:
```suggestion
    loop and switch and transfer control to the parent operation. It is only
```

I think?

Reply:
I don't think so. We're either breaking out of a loop or a switch operation. After breaking out of the loop or switch, we're transferring control to the parent operation. Would this be better?
```suggestion
    loop or switch operation and transfer control to the parent operation. It is only
```

Reply:
Oh, yeah, that was a thinko/misread from me.  I like the 'operation' word being there as I think it is an improvement, else LGTM.